save-backend/src/test/kotlin/com/saveourtool/save/backend/controller/AgentsControllerTest.kt

```diff
@@ -68,7 +68,7 @@ class AgentsControllerTest {
             .accept(MediaType.APPLICATION_JSON)
             .bodyValue(
                 listOf(
-                    AgentStatusDto(LocalDateTime.now(), AgentState.IDLE, "container-1")
+                    AgentStatusDto(AgentState.IDLE, "container-1")
```

Comment:
Time is not needed on AgentStatus? Why?

Reply:
it's set as LocalDateTime.now() in constructor. We set only in two places another values and it's a test